### PR TITLE
Add object access log

### PR DESCRIPTION
**Pull request by Retler**
### Description
This PR enriches some of our log statements with an object ID. This enables us to trace access to objects and permissions. Note that the tracing information is stored in elasticsearch rather than the database as originally intended.

### Parent Issue
Issue #46 

### Comments for the Reviewers
- Check that we have all the information we need to track users access to objects and permissions. This branch is currently deployed on staging, so elasticsearch should contain the updated logs.
- I have attached a small example from Kibana on how we can view access to objects with current implementation.

### Type(s) of Change (Split the PR if you check many)
- [ ] Added user feature
- [x] Added technical feature
- [ ] Added tests
- [ ] Refactor
- [ ] Bugfix
- [ ] Updated documentation
- [ ] Infrastructure changes (Terraform, GCP, K8s, other)
- [ ] CI/CD workflow changes

### General Checklist
- [x] I have pulled in the latest changes from master.
- [x] I have run `make lint`.
- [x] I have successfully run `make docker-up; make tests`.
- [ ] I have updated relevant workflows and deployment tools.
- [ ] I have updated/added relevant documentation (readme, doc comments, etc).
- [ ] I have added the license to new source code files.
- [x] I have spent some time looking over the full diff before creating this PR.

### CODE REVIEW

**Comment by Retler** [comment]
Example of object access log with current implementation:
object_log

File: encryption-service/app/permissions_handler.go [hunk 1]
```diff
@@ -100,6 +103,9 @@ func (app *App) AddPermission(ctx context.Context, request *AddPermissionRequest
 		return nil, status.Errorf(codes.Internal, "error encountered while adding permission")
 	}
 
+	ctx = context.WithValue(ctx, contextkeys.ObjectIDCtxKey, oid)
+	log.Info(ctx, "AddPermission: Permission added")
```
Comment (philip-cc) [inline encryption-service/app/permissions_handler.go:107]: Should we log the UID that was added (respectively removed) here?

Comment (Retler) [reply encryption-service/app/permissions_handler.go:108]: fixed

**Review by philip-cc — APPROVED** [review]
Ship it!

**Pull request merged**